app/src/main/java/io/neurolab/activities/PinLayoutActivity.java

```diff
@@ -186,4 +191,18 @@ private void midPoint(PointF point, MotionEvent event) {
         } catch (Exception e) {/**/}
         point.set(x / 2, y / 2);
     }
+
+    @Override
+    public boolean onSupportNavigateUp(){
+        onBackPressed();
+        return true;
+    }
+
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        StatisticsFragment.parsedData = null;
```

Comment:
What is this in PinLayout screen?

Reply:
@jddeep  That was a misktake, Sorry about that!
Removing it!